common/app/cards/CardType.scala

```diff
@@ -4,6 +4,8 @@ sealed trait CardType {
   val cssClassName: String
 
   def showVideoPlayer = this match {
+      // TODO check this is really correct - only should be playable if they're big enough - Third could be playable on desktop, but at mobile it renders them wide, narrow, narrow above each other
+      // so only the top one makes sense to be playable
```

Comment:
Mobile + Tablet card sizes can be mixed willy nilly (although there are probably only a small set of actual combinations). This parameter is to say whether a video player appears at *any* breakpoint. If it does, then we needed previously to show the video tag.